data/data_loader_homo.py

```diff
@@ -46,27 +76,53 @@ def create_dataloaders_homo(
     customer_id_map = read_json("data/derived/customer_id_map_forward.json")
     article_id_map = read_json("data/derived/article_id_map_forward.json")
 
-    return (
-        NeighborLoader(
+    train_loader = shuffle_data(
+        LinkNeighborLoader(
             train_split,
             batch_size=config.batch_size,
             num_neighbors=[10, 10],
-            shuffle=True,
-        ),
-        NeighborLoader(
-            val_split,
-            batch_size=config.batch_size,
-            num_neighbors=[10, 10],
-            shuffle=True,
-        ),
-        NeighborLoader(
-            test_split,
-            batch_size=config.batch_size,
-            num_neighbors=[10, 10],
-            shuffle=True,
-        ),
+            # shuffle=True, # This is not yet implemented in the source code
+            directed=False,
+            edge_label_index=train_split.edge_label_index,
+            edge_label=train_split.edge_label,
+        )
+    )
+
+    val_loader = shuffle_data(
```

Comment:
you mentioned that we may need to do shuffle_data on the dataset, not on the dataloader